addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2958,6 +2958,22 @@ options.registry.HideFooter = VisibilityPageOptionUpdate.extend({
     shownValue: 'shown',
 });
 
+options.registry.ContainerWidthFooter = options.registry.ContainerWidth.extend({
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+    /**
+     * Allow to customize the view only if the footer copyright section
+     * is the target (due to the data-apply-to)
+     * @override
+     */
+    customizeWebsiteViews: async function (previewMode, widgetValue, params) {
+        if (this.$target[0].closest(".o_footer_copyright")) {
+            await this._super(...arguments);
+        }
```

Comment:
It's not critical but I think it could be better:
- Set width to full -> page reload, it looks good
- Disable copyright -> page reload, copyright is gone, good
- Set width to small -> page reload (that's actually not necessary, the copyright is not enabled but that's ok), it looks good
- Re-enable copyright -> page reload... and you have a full copyright but a small footer.

It's probably easy enough to fix :+1: